src/dbnode/persist/fs/seek_manager_test.go

```diff
@@ -139,6 +148,10 @@ func TestSeekerManagerUpdateOpenLease(t *testing.T) {
 		require.Equal(t, 1, seekers.active.volume)
 		byTime.RUnlock()
 	}
+	// Ensure that the old seekers actually get closed.
+	mockSeekerStatsLock.Lock()
+	require.Equal(t, len(shards)*defaultFetchConcurrency, numMockSeekerCloses)
+	mockSeekerStatsLock.Unlock()
```

Comment:
Does this test also catch the case where no leases are borrowed?

Reply:
Yeah that test is specifically testing when there are no seekers borrowed, the concurrency test tests for when they *are* borrowed